### PR TITLE
Neighborhood class

### PR DESCRIPTION
**Pull request by tgustafson2**
Implements #16

### CODE REVIEW

**Review by dukeph0 — APPROVED** [review]
Looks good. needs server and client to be running through

File: server/neighborhood.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        if(info[10]>info[11]&&info[10]>info[12]&&info[10]>info[13]){
+            this._closest_metro="Los Angeles";
+        }
+        else if(info[11]>info[10]&&info[11]>info[12]&&info[11]>info[13]){
+            this._closest_metro="San Diego";
+        }
+        else if(info[12]>info[11]&&info[12]>info[10]&&info[12]>info[13]){
+            this._closest_metro="San Jose";
+        }
+        else if(info[13]>info[11]&&info[13]>info[12]&&info[13]>info[10]){
```
Comment (raymondwzeng) [inline server/neighborhood.js:26]: Interesting. Didn't catch this the first time around. Should we cache this information or have it as a getter? Doesn't matter really to me.

Comment (tgustafson2) [reply server/neighborhood.js:26]: I was just thinking that way there was less computation for each function call of that

**Comment by tgustafson2** [comment]
Fixed failing test issues

**Comment by raymondwzeng** [comment]
Did you push any changes? I don't see a difference in the codebase.

**Comment by tgustafson2** [comment]
I guess not, I thought I did, my bad. It should be there now

**Review by raymondwzeng — APPROVED** [review]
Looks good to me!

**Pull request merged**